File: spaceship_crm/java8/spaceship_crm_java8_jsp_jpa/src/main/java/com/spaceship/crm/bean/UserActivityBean.java
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package com.spaceship.crm.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserActivityBean {
    private Long userId;
    private String firstName;
    private String lastName;

    private String activityName;

    private Date start;
    private Date end;
    private Date added;
}

File: spaceship_crm/kotlin/spaceship_crm_kotlin_spring_mustache_jpa/src/main/kotlin/com/spaceship/crm/bean/UserActivityBean.kt
/**
 * Author veni.vidi.dev (veni.vidi.dev@gmail.com)
 */
package com.spaceship.crm.bean

import java.util.*

class UserActivityBean(
        var userId: Long? = null,
        var firstName: String? = null,
        var lastName: String? = null,
        var activityId: Long? = null,
        var activityName: String? = null,
        var start: Date? = null,
        var end: Date? = null,
        var added: Date? = null
)